mixins/form.jsx

```diff
@@ -96,7 +95,7 @@ module.exports = {
     this.updateHeight();
   },
   submit: function(action, props, callback) {
-    props.localeCode = this.state.localeCode || "US";
+    props.locale = this.props.locales[0];
```

Comment:
To add more context.

I'm just passing along the locale for anyone on the other side (just our code) to handle with how it wants to.

This means it's up to us to filter out "en-US" to "US" where it makes sense.